core/iwasm/common/wasm_application.c

```diff
@@ -100,18 +100,15 @@ wasm_application_execute_main(WASMModuleInstanceCommon *module_inst, int32 argc,
         return wasm_runtime_create_exec_env_and_call_wasm(module_inst, func, 0,
                                                           NULL);
     }
-    else {
-        wasm_runtime_set_exception(
-            module_inst, "lookup _start function failed, may run with \'-f\'");
-        return false;
-    }
 #endif /* end of WASM_ENABLE_LIBC_WASI */
 
     if (!(func = wasm_runtime_lookup_function(module_inst, "main", NULL))
         && !(func = wasm_runtime_lookup_function(module_inst,
                                                  "__main_argc_argv", NULL))
         && !(func = wasm_runtime_lookup_function(module_inst, "_main", NULL))) {
-        wasm_runtime_set_exception(module_inst, "lookup main function failed");
+        wasm_runtime_set_exception(
+            module_inst,
+            "lookup the entry point symbol(like _start, _main, _main) failed");
```

Comment:
How about "lookup the entry point symbol (e.g. main, _main, __main_argc_argv) failed", _start isn't to be find here, and "_main, _main" is duplicated

Reply:
plan to include the *else* case of L99 missing the `_start`? maybe we shall create two messages when 
- `WASM_EABLE_LIBC_WASI=1`. *lookup the entry point symbol (like _start, main, _main, __main_argc_argv) failed*
- `WAMS_ENABLE_LIBC_WASI=0`. *lookup the entry point symbol (like main, _main, __main_argc_argv) failed*